src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/OrchardCore.Application.Cms.Core.Targets.csproj

```diff
@@ -24,6 +24,7 @@
   -->
 
   <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore.Modules\OrchardCore.Placements\OrchardCore.Placements.csproj" />
```

Comment:
```
    <!-- 
      When adding a reference to this list, please keep it ordered alphabetically, and set PrivateAssets="none" 
    -->
```

Hmm, do we need to move this comment so it gets seen ;)

Reply:
Sorry fot that. I just used the VS UI to add the project reference.